tenets/codelingo/code-review-comments/use-crypto-rand/codelingo.yaml

```diff
@@ -0,0 +1,39 @@
+funcs:
+  - name: isKeyName
+    type: asserter
+    body: |
+      function(name) {
+        return name.toLowerCase().indexOf("key") !== -1
+      }
+tenets:
+  - name: use-crypto-rand
+    flows:
+      codelingo/review:
+        comment: |
+          Do not use package math/rand to generate keys, even 
+          throwaway ones. Unseeded, the generator is completely predictable. 
+          Seeded with time.Nanoseconds(), there are just a few bits of entropy. 
+          Instead, use crypto/rand's Reader, and if you need text, print to 
+          hexadecimal or base64
+    query: |
+      import codelingo/ast/go
+
+      go.import_spec(depth = any):     
+        go.basic_lit:
+          value == "math/rand"
```

Comment:
@Daanikus catch the case where the import is renamed:

```go
import myRand "math/rand"
```

 Add a test case for it.